### ISSUE TITLE
Glass ocean not working

### ISSUE THREAD

**Issue by NCDyoshii** [cleannrooster/Spellblade-Next#21]
Unsure if the mod being included with a modpack is affecting anything but anytime I try to load into the glass ocean, the world sort of breaks. Its still saying I'm in the overworld but nothing around me is loaded in. No water, no buildings, nothing. The only thing consistent is the wizards spawning to kill me. I have no idea how this happened as just a couple of days ago the dimension was working just fine but once I tried to go in with my friends it would never work for me.

**Comment by NCDyoshii** [comment]
![image_2024-03-20_222356989](https://github.com/cleannrooster/Spellblade-Next/assets/164285970/34c2c451-257c-459f-baba-cc7030926902)
![image_2024-03-20_222410069](https://github.com/cleannrooster/Spellblade-Next/assets/164285970/2b01683b-5bf1-450e-ba61-495d61a85ce4)
![image_2024-03-20_222419017](https://github.com/cleannrooster/Spellblade-Next/assets/164285970/ab6d3e1a-5744-45a2-b46b-7cafc6ac00fd)
![image_2024-03-20_222427800](https://github.com/cleannrooster/Spellblade-Next/assets/164285970/05cc767c-e2cf-4252-8bd9-d395886196e3)

**Comment by cleannrooster** [comment]
Is there any sort of error message?

**Comment by NCDyoshii** [comment]
No, not a single error message, it just kinda refuses to load the dimension without saying anything
